docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc

```diff
@@ -1,26 +1,56 @@
-= SBOMs
+= Inspecting SBOMs
 
 :toc: left
 :icons: font
-:numbered:
 :source-highlighter: highlightjs
 
-== Software bill of materials
-A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM is a list of all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries might enable specific functionality, facilitate development, or serve some other purpose. 
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose. 
 
-You can use an SBOM to better understand the composition of your software, identify vulnerabilities, and assess the potential impact of any security issues that might arise. You might also need to provide your customers with an SBOM to comply with cybersecurity regulations. 
+You can use an SBOM to better understand your software’s composition, identify vulnerabilities, and assess the potential impact of any security issues that may arise. You may also need to provide your customers with a component’s SBOM to comply with cybersecurity regulations. 
 
-== Downloading an SBOM
+== Downloading an SBOM in the web UI
 
-*Prerequisites*
+.Prerequisites
+
+* You must have an application that {ProductName} has successfully built and deployed.
+
+.Procedure
+//Replace all images here
+Complete the following steps to download a component's SBOM:
+
+. In the console, navigate first to the *Activity* tab, then to the *Pipeline runs* tab.
++
+image::sbom/activity-to-pipeline-runs.png[alt=Pipeline runs tab] 
```

Comment:
I'm not a fan of including screenshots in our documentation. I see the reasoning behind it, so I'm not totally against it. 

But I think screenshots can be difficult to maintain and difficult to ensure accessibility. We also provide a lot of in-app documentation, so I don't think we need to include screenshots in our doc set. 

In short, my vote is to remove them.